openapi.yml

```diff
@@ -146,19 +163,69 @@ components:
       format: FiscalCode
       x-import: italia-ts-commons/lib/strings
       example: SPNDNL80R13C555X
+    ArrayIndexUpTo4:
+      type: integer
+      minimum: -1
+      maximum: 5
+      exclusiveMaximum: true
+    ExistingArrayIndexUpTo4:
+      type: integer
+      minimum: 0
+      maximum: 5
+      exclusiveMaximum: true
+    NonEmptyString:
+      type: string
+      format: NonEmptyString
+      x-import: italia-ts-commons/lib/strings
     OrganizationFiscalCode:
       type: string
       description: Organization's fiscal code.
       format: OrganizationFiscalCode
       x-import: italia-ts-commons/lib/strings
       example: 01234567890
+    OrganizationScope:
+      type: string
+      enum:
+        - "LOCAL"
+        - "NATIONAL"
     UserRole:
       type: string
       enum:
         - "ORG_DELEGATE"
         - "ORG_MANAGER"
         - "DEVELOPER"
         - "ADMIN"
+    LegalRepresentative:
+      type: object
+      properties:
+        email:
+          $ref: "#/components/schemas/EmailAddress"
+        familyName:
+          $ref: "#/components/schemas/NonEmptyString"
+        firstName:
+          $ref: "#/components/schemas/NonEmptyString"
+        fiscalCode:
+          $ref: "#/components/schemas/FiscalCode"
+        phoneNumber:
+          $ref: "#/components/schemas/NonEmptyString"
+        role:
+          $ref: "#/components/schemas/UserRole"
+      required:
+        - email
+        - familyName
```

Comment:
note that we have used snake_cased properties in other part of io project, 
maybe it's worth to follow that convention here as well ?

@cloudify @ioggstream  ?

Reply:
I'm personally for snake-case. It's up to you to decide, as long as it's consistent inside the API.

Reply:
In the code on this repository, I only used camel-cased names for properties, because I used the same style suggested by lint rules for variable names, so there is consistency.

Reply:
@gunzip so if there's a standard for all IO API I'd place the rule in the linter first

Reply:
@gunzip indeed, all our APIs have snake case - it's described in the handbook, [item 6 of general guidelines](https://github.com/teamdigitale/io-handbook/blob/master/development-guidelines.md#general-guidelines), [here's the JSON guidelines](https://opensource.zalando.com/restful-api-guidelines/#json-guidelines)